salt/cloud/clouds/vmware.py

```diff
@@ -2350,6 +2393,33 @@ def create(vm_):
             )
             clone_spec.customization = custom_spec
 
+        if customization and (devices and 'network' in list(devices.keys())) and 'Windows' in object_ref.config.guestFullName:
+            global_ip = vim.vm.customization.GlobalIPSettings()
+
+            if 'dns_servers' in list(vm_.keys()):
+                global_ip.dnsServerList = vm_['dns_servers']
+
+            identity = vim.vm.customization.Sysprep()
+            identity.guiUnattended = vim.vm.customization.GuiUnattended()
+            identity.guiUnattended.autoLogon = False
+            identity.guiUnattended.password = vim.vm.customization.Password()
+            identity.guiUnattended.password.value = vm_['win_password']
+            identity.guiUnattended.password.plainText = True
```

Comment:
Let the user specify if they want to specify the password as plain text or encrypted? Default should be encrypted perhaps and they can switch to plain-text by specifying ``plain-text: True`` ?